var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -141,6 +146,25 @@ class Cmake(Package):
 
     phases = ['bootstrap', 'build', 'install']
 
+    @staticmethod
```

Comment:
I think each class could choose either. For class methods the class is provided as an additional initial argument. If the method implementation doesn't refer to class details, then it can use a `staticmethod`. Generally though classes probably will refer to such details though (e.g. the regexes used to find executables) so to set an example, I will change it to a `classmethod`.